fsspec/implementations/tests/test_prefix.py

```diff
@@ -0,0 +1,198 @@
+from __future__ import absolute_import, division, print_function
+
+import os
+import os.path
+import tempfile
+from contextlib import contextmanager
+from pathlib import Path
+
+import pytest
+
+import fsspec
+from fsspec.core import OpenFile
+from fsspec.implementations.local import make_path_posix
+from fsspec.implementations.prefix import PrefixFileSystem
+
+files = {
```

Comment:
This and the contexts are copied from somewhere?

Reply:
Seem more like a fixture than a context

Reply:
Yep, this was copied before from `test_local`. Didn't touch it much, as I believe we need to rework these tests and introduce common fsspec fixtures. At least it is isolated from `test_local` for now.

For prefixfs, some unit tests that would make sure that the path is properly joined and properly passed to the underlying filesystem should be enough (plus some for transforming stuff back, e.g. `ls`). So I plan on introducing those along with support for new methods in the follow-up.